src/theme/DocItem/TOC/Mobile/styles.module.scss

```
@@ -0,0 +1,153 @@
+@use "../../../../css/functions" as *;
+
+// Paired with the .sidebar column in DocItem/Layout, which takes over at 1200px.
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)